include/poller.php

```diff
@@ -421,6 +459,10 @@ function poller_kill_stale_workers() {
 /**
  * @brief Checks if the number of active workers exceeds the given limits
  *
+ * @param integer $entries The number of not executed entries in the worker queue
+ * @param integer $top_priority The highest not executed priority in the worker queue
+ * @param boolean $high_running Is a process with priority "$top_priority" running?
+ *
  * @return bool Are there too much workers running?
```

Comment:
The signature in line 426 does not have such parameter. Maybe accidently added it here or missed there?